pkg/apis/core/v1beta1/constants/types_constants.go

```diff
@@ -398,6 +398,10 @@ const (
 	LabelLogging = "logging"
 	// LabelMonitoring is a constant for a label for monitoring stack configurations
 	LabelMonitoring = "monitoring"
+	// LabelKeyCustomLoggingResource is a label key used for the logging custom resources
+	LabelKeyCustomLoggingResource = "fluentbit.gardener/type"
```

Comment:
I followed the discussion above. Let's add a TODO, that this label key has to be migrated to `fluentbit.gardener.cloud/type`:

```suggestion
	// TODO(Kristian-ZH): the label key has to be migrated to "fluentbit.gardener.cloud/type".
	LabelKeyCustomLoggingResource = "fluentbit.gardener/type"
```